src/main/webapp/app/shared/sidebar/sidebar-card-item/sidebar-card-item.component.html

```diff
@@ -58,6 +64,9 @@
                         <fa-icon [fixedWidth]="true" [icon]="sidebarItem.rightIcon" />
                     }
                 </span>
+                @if (unreadCount && unreadCount > 0) {
+                    <span class="unread-count">{{ formattedUnreadCount }}</span>
```

Comment:
I would decrease the spacing between the heart and the counter: `me-n2`
In addition I would also add `px-2` to the button of the `faEllipsisVertical `icon in the `conversation-options.component.html` file